cost/azure/hybrid_use_benefit/README.md

```diff
@@ -2,13 +2,14 @@
 
 ## What it does
 
-This Policy Template is used to automatically apply the Azure Hybrid Use Benefit (AHUB) to all eligible VMs in an Azure Subscription.
+This Policy Template is used to automatically apply the Azure Hybrid Use Benefit (AHUB) to all eligible VMs in an Azure Subscription and provides a monthly savings value if AHUB is enabled.
 
 ## Functional Details
 
-- The policy identifies all Windows server instances that are not currently using [Azure Hybrid Use Benefit](https://azure.microsoft.com/en-us/pricing/hybrid-benefit/). It raises an incident for all applicable VMs not currently using AHUB, which once approved, will enable AHUB on all identified instances.1
+- The policy identifies all Windows server instances that are not currently using [Azure Hybrid Use Benefit](https://azure.microsoft.com/en-us/pricing/hybrid-benefit/). It raises an incident for all applicable VMs not currently using AHUB and provides a monthly savings amount if AHUB is enabled, which once approved, will enable AHUB on all identified instances.
 - The Exclusion Tag parameter is a string value. Supply the Tag Key only. Tag Values are not analyzed and therefore are not need. If the exclusion tag key is used on an Instance, that Instance is presumed to be exempt from this policy.
 - This policy does not track licenses or availability. It is your responsibility to ensure you are not under licensed.
+- The savings currency is provided in $USD, and currency conversion is not supported.
```

Comment:
We should add support for this. We now have an API capable of delivering this. See this policy: https://raw.githubusercontent.com/flexera-public/policy_templates/POL-832-aws-unused-ip-audit/cost/aws/unused_ip_addresses/aws_unused_ip_addresses.pt

Specifically ds_currency_target and the following datasources. You should be able to basically copypaste this into this policy and tweak it to match what this policy is doing.